CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.7.0] - 2021-05-19
```

Comment:
Just please change it to following (the  specific version and date will be added automatically during release process)
```md
## [Unreleased]
```

Reply:
Fixed